model/work.py

```diff
@@ -101,6 +101,8 @@ class Work(Base):
     CURRENTLY_AVAILABLE = "currently_available"
     ALL = "all"
 
+    DEFAULT_IDENTIFIER_CUTOFF = 100
```

Comment:
I don't think you used this, you have another default constant in another place.

Reply:
Yeah, this was the first thing I wrote before I figured it made more sense to put these things into PresentationCalculationPolicy.